install_ffmpeg.sh

```diff
@@ -117,7 +117,7 @@ fi
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
   git clone https://github.com/livepeer/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
   cd "$HOME/ffmpeg"
-  git checkout 3ea705767720033754e8d85566460390191ae27d
```

Comment:
Can we still do a specific commit hash here? I like that it makes it impossible for someone to compromise `https://github.com/livepeer/ffmpeg` and insert their own code into the process.

Reply:
Makes sense, aligns with us using exact commit hashes for LPMS module in go-livepeer too, and keeping things explicit. Will fix once https://github.com/livepeer/FFmpeg/pull/2 is merged and commit ID is finalized!

Reply:
Updated to use the commit hash in cb944d1